app/components/draft_scheduled_post_header/profile_avatar.tsx

```diff
@@ -20,12 +20,12 @@ type Props = {
 const styles = StyleSheet.create({
     avatarContainer: {
         backgroundColor: 'rgba(255, 255, 255, 0.4)',
-        width: 24,
-        height: 24,
+        width: 20,
+        height: 20,
```

Comment:
I did ask but dont remember the answer, why the change in size?

Reply:
Since we use a common component for both drafts and scheduled posts, the previous implementation did not match the Figma design (or was left in the draft pr), which is why the changes were made.